typescript/sdk/src/ism/EvmIsmReader.ts

```diff
@@ -230,8 +238,9 @@ export class EvmIsmReader implements IsmReader {
 
     // if it has VERIFIED_MASK_INDEX, it's AbstractMessageIdAuthorizedIsm which means OPStackIsm
     const opStackIsm = OPStackIsm__factory.connect(address, this.provider);
+    this.assertModuleType(await opStackIsm.moduleType(), ModuleType.NULL);
```

Comment:
@paulbalaji - pulled this guy out of the try catch as I'm not sure we want to catch it with that error message

Reply:
i'd say to solve both the comments here do something like this just once right at the top of the deriveNullConfig function, so we do the null assert once at the top and then we go into checking which null ism it is

```ts
    const ism = IInterchainSecurityModule__factory.connect(address, this.provider);
    this.assertModuleType(await ism.moduleType(), ModuleType.NULL);
```